scripts/neuro-spicy-init.ps1

```
@@ -201,6 +201,23 @@ function Install-Dependencies {
         Write-ColorOutput "✅ Python already installed: $pythonVersion" "Green"
     }
     
+    # Check for OpenClaw
+    if (-not (Test-Command "openclaw")) {
+        $response = Read-Host "📦 OpenClaw not found. Install? (y/N)"
+        if ($response -match "^[Yy]$") {
+            if (Test-Command "npm") {
+                npm install -g openclaw@latest
+            } else {
+                iwr -useb https://openclaw.ai/install.ps1 | iex
```

PSScriptAnalyzer (1.24.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)